commands/export_e2e_test.go

```diff
@@ -383,3 +383,66 @@ func (s *MmctlE2ETestSuite) TestExportJobListCmdF() {
 		s.Require().Equal(job2, printer.GetLines()[1].(*model.Job))
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestExportJobCancelCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.Run("Cancel an export job without permissions", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		job, appErr := s.th.App.CreateJob(&model.Job{
+			Type: model.JobTypeExportProcess,
+		})
+		s.Require().Nil(appErr)
+
+		time.Sleep(time.Millisecond)
+
+		err := exportJobCancelCmdF(s.th.Client, cmd, []string{job.Id})
+		s.Require().EqualError(err, "failed to get export job: : You do not have the appropriate permissions., ")
+		s.Require().Empty(printer.GetLines())
+		s.Require().Empty(printer.GetErrorLines())
+	})
+
+	s.RunForSystemAdminAndLocal("No export jobs to cancel", func(c client.Client) {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		err := exportJobCancelCmdF(c, cmd, []string{model.NewId()})
+		s.Require().EqualError(err, "failed to get export job: : Unable to get the job., ")
+		s.Require().Empty(printer.GetLines())
+		s.Require().Empty(printer.GetErrorLines())
+	})
+
+	s.RunForSystemAdminAndLocal("Cancel an export job", func(c client.Client) {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		job1, appErr := s.th.App.CreateJob(&model.Job{
+			Type: model.JobTypeExportProcess,
+		})
+		s.Require().Nil(appErr)
+
+		time.Sleep(time.Millisecond)
+
+		job2, appErr := s.th.App.CreateJob(&model.Job{
+			Type: model.JobTypeExportProcess,
+		})
+		s.Require().Nil(appErr)
+
+		err := exportJobCancelCmdF(c, cmd, []string{job1.Id})
+		s.Require().Nil(err)
+		s.Require().Empty(printer.GetLines())
+		s.Require().Empty(printer.GetErrorLines())
+
+		// Get jobs again to refresh their status
+		job1, _ = s.th.App.GetJob(job1.Id)
+		job2, _ = s.th.App.GetJob(job1.Id)
```

Comment:
We need to check errors here. It will `panic` in case of `nil` values return from these calls.

Reply:
Thank you for the note! I'll make sure to add checking for error == nil. I just realized that I accidentally wrote job1 retrieval 2 times so I have to fix that too.